### DISCUSSION TITLE
It has an error like missing Plugins, how to download it  ?

### DISCUSSION THREAD

**Discussion by Cucurbit1988** [ToolTech/Streaming_Map_Demo#7] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32631951/160867906-a5789d49-db26-43dc-818b-366ce89c2d5b.png)
I really like your project

**Comment by ToolTech** [comment, marked as answer]
First you need to install the libs from nuget.org

You open the dev studio solution in InstallPackages/InstallPackages.sln

Select a build platform like x64 or ARM64 depending on what subsystem you target. Right now only android ARM and ARM64 as well as Windows x64 are supported for this demo.

You select a debug or release configuration and then you just build the solution. This shall put the latest libs under Assets

**Reply by Cucurbit1988** [reply]
This is the first time I ask a question on Github and someone replies. Thank you very much
